Mathlib/MeasureTheory/Integral/Layercake.lean

```diff
@@ -377,3 +381,95 @@ theorem lintegral_rpow_eq_lintegral_meas_lt_mul (μ : Measure α) [SigmaFinite 
 #align lintegral_rpow_eq_lintegral_meas_lt_mul lintegral_rpow_eq_lintegral_meas_lt_mul
 
 end LayercakeLT
+
+section LayercakeIntegral
+
+variable {α : Type*} [MeasurableSpace α] {μ : Measure α} [SigmaFinite μ]
+
+/-- If `f` is integrable, then for any `c > 0` the set `{x | ‖f x‖ ≥ c}` has finite measure. -/
+lemma Integrable.measure_const_le_norm_lt_top {E : Type*} [NormedAddCommGroup E] [MeasurableSpace E]
+    [BorelSpace E] {f : α → E} (f_intble : Integrable f μ) {c : ℝ} (c_pos : 0 < c) :
```

Comment:
You shouldn't need to have a measurable space structure on `E` in the statement. If you need it in the proof, you can use `borelize E`. Also, you shouldn't need mu to be sigma-finite.